commit 7fd9fd94fb8f23b86578ed86906b8caf0417db82
Merge: 5fa0f5238b d9589d4051
Author: REDACTED <REDACTED>
Date:   Wed Dec 4 23:06:31 2019 +0100

    Sync with 2.22.2
    
    * maint-2.22: (43 commits)
      Git 2.22.2
      Git 2.21.1
      mingw: sh arguments need quoting in more circumstances
      mingw: fix quoting of empty arguments for `sh`
      mingw: use MSYS2 quoting even when spawning shell scripts
      mingw: detect when MSYS2's sh is to be spawned more robustly
      t7415: drop v2.20.x-specific work-around
      Git 2.20.2
      t7415: adjust test for dubiously-nested submodule gitdirs for v2.20.x
      Git 2.19.3
      Git 2.18.2
      Git 2.17.3
      Git 2.16.6
      test-drop-caches: use `has_dos_drive_prefix()`
      Git 2.15.4
      Git 2.14.6
      mingw: handle `subst`-ed "DOS drives"
      mingw: refuse to access paths with trailing spaces or periods
      mingw: refuse to access paths with illegal characters
      unpack-trees: let merged_entry() pass through do_add_entry()'s errors
      ...

commit d9589d4051537c387b70dc76e430c61b4c85a86d
Author: REDACTED <REDACTED>
Date:   Wed Dec 4 23:05:10 2019 +0100

    Git 2.22.2
    
    Signed-off-by: REDACTED <REDACTED>

diff --git a/Documentation/RelNotes/2.22.2.txt b/Documentation/RelNotes/2.22.2.txt
new file mode 100644
index 0000000000..940a23f0d9
--- /dev/null
+++ b/Documentation/RelNotes/2.22.2.txt
@@ -0,0 +1,8 @@
+Git v2.22.2 Release Notes
+=========================
+
+This release merges up the fixes that appear in v2.14.6, v2.15.4,
+v2.17.3, v2.20.2 and in v2.21.1, addressing the security issues
+CVE-2019-1348, CVE-2019-1349, CVE-2019-1350, CVE-2019-1351,
+CVE-2019-1352, CVE-2019-1353, CVE-2019-1354, CVE-2019-1387, and
+CVE-2019-19604; see the release notes for those versions for details.
diff --git a/GIT-VERSION-GEN b/GIT-VERSION-GEN
index ac51bac07b..a35956a72f 100755
--- a/GIT-VERSION-GEN
+++ b/GIT-VERSION-GEN
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 GVF=GIT-VERSION-FILE
-DEF_VER=v2.22.1
+DEF_VER=v2.22.2
 
 LF='
 '
diff --git a/RelNotes b/RelNotes
index 30cbde7a54..6c075ba53f 120000
--- a/RelNotes
+++ b/RelNotes
@@ -1 +1 @@
-Documentation/RelNotes/2.22.1.txt
\ No newline at end of file
+Documentation/RelNotes/2.22.2.txt
\ No newline at end of file
